### PR TITLE
More Synth tweaks

### PR DESCRIPTION
**Pull request by DeltaFire15**
## About The Pull Request
This PR does the following:
Synths can now be revived via surgery, (emergency reboot surgery), which requires no defib (they still need to be in a defibbable state though)
Synths do no get hardstunned from EMPs anymore.
Synths have access to the embed removal surgery again.
**Reduces the capped damage on synth limbs when past the damage threshold to 10 (from 15).**

## Why It's Good For The Game
More synth balance stuff adressing their issues.

## Changelog
:cl:
add: A new surgery, allowing revival of synths without a defib at hand.
balance: Semi-permanent damage of Synth limbs caused by passing the damage threshold: 10 <- 15.
tweak: The embed removal surgery now has a version for Synths.
balance: EMPs no longer hardstun Synths.
/:cl:

### CODE REVIEW

**Comment by TripleShades** [comment]
the fuck do you mean they dont need a defib
does this also apply to IPCs or are you just using synth to refer to both?

**Comment by TripleShades** [comment]
Does this also fix the brain surgery not actually removing brain damage? Or is that going to be seperate?

**Comment by DeltaFire15** [comment]
> the fuck do you mean they dont need a defib
> does this also apply to IPCs or are you just using synth to refer to both?

'Synth' in this case means 'any species that is a robot but if I say robot people will think of borgs'. If I talk about the lizards in specific, I refer to them as 'Synthliz' or 'Synthlizard'

**Comment by DeltaFire15** [comment]
> Does this also fix the brain surgery not actually removing brain damage? Or is that going to be seperate?

Does the brain surgery for robots not work? I've never actually heard that before.

**Comment by TripleShades** [comment]
> > Does this also fix the brain surgery not actually removing brain damage? Or is that going to be seperate?
> 
> Does the brain surgery for robots not work? I've never actually heard that before.

yes it does not work, it was reported previously and supposedly fixed, but it still doesn't remove brain trauma properly

**Comment by LordSaladin** [comment]
nice, can't we just smack a reboot board on em?

**Comment by DeltaFire15** [comment]
> > > Does this also fix the brain surgery not actually removing brain damage? Or is that going to be seperate?
> > 
> > 
> > Does the brain surgery for robots not work? I've never actually heard that before.
> 
> yes it does not work, it was reported previously and supposedly fixed, but it still doesn't remove brain trauma properly

Just tested it, it seems to work fine, both for brain damage repair aswell as removing brain traumata. Frankly, unsure what your problem with it is.

**Comment by TripleShades** [comment]
> > > > Does this also fix the brain surgery not actually removing brain damage? Or is that going to be seperate?
> > > 
> > > 
> > > Does the brain surgery for robots not work? I've never actually heard that before.
> > 
> > 
> > yes it does not work, it was reported previously and supposedly fixed, but it still doesn't remove brain trauma properly
> 
> Just tested it, it seems to work fine, both for brain damage repair aswell as removing brain traumata. Frankly, unsure what your problem with it is.

repeatedly have gotten brain trauma that messes up speech to be mixed up, and the surgery did not fix it even after multiple attempts. this has also happened to other players with that specific trauma.

**Comment by DeltaFire15** [comment]
> > > > > Does this also fix the brain surgery not actually removing brain damage? Or is that going to be seperate?
> > > > 
> > > > 
> > > > Does the brain surgery for robots not work? I've never actually heard that before.
> > > 
> > > 
> > > yes it does not work, it was reported previously and supposedly fixed, but it still doesn't remove brain trauma properly
> > 
> > 
> > Just tested it, it seems to work fine, both for brain damage repair aswell as removing brain traumata. Frankly, unsure what your problem with it is.
> 
> repeatedly have gotten brain trauma that messes up speech to be mixed up, and the surgery did not fix it even after multiple attempts. this has also happened to other players with that specific trauma.

I assume thats not actually a trauma, but instead the effects of residual braindamage that remains.

**Comment by TripleShades** [comment]
> > repeatedly have gotten brain trauma that messes up speech to be mixed up, and the surgery did not fix it even after multiple attempts. this has also happened to other players with that specific trauma.
> 
> I assume thats not actually a trauma, but instead the effects of residual braindamage that remains.

it's regardless removed with brain surgery on non-synthetic crew, so why would it be different here?

**Comment by necromanceranne** [comment]
> nice, can't we just smack a reboot board on em?

You could even use the reboot board as the tool for the defibrillation step.

**Comment by DeltaFire15** [comment]
> > nice, can't we just smack a reboot board on em?
> 
> You could even use the reboot board as the tool for the defibrillation step.

.. That is actually a really good idea, tweaked it so both multitool and borg reboot module are accepted by the step.

File: code/modules/surgery/emergency_reboot.dm [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		var/policy = late? policies[POLICYCONFIG_ON_DEFIB_LATE] : policies[POLICYCONFIG_ON_DEFIB_INTACT]
+		if(policy)
+			to_chat(target, policy)
+		target.log_message("revived using surgical revival, [tplus] deciseconds from time of death, considered [late? "late" : "memory-intact"] revival under configured policy limits.", LOG_GAME)
+		return TRUE
+	else
+		target.visible_message("...[target]'s posibrain flickers a few times, before the lights fade yet again...")
+		return FALSE
+
+/datum/surgery_step/revive/failure(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
```
Comment (Linzolle) [inline code/modules/surgery/emergency_reboot.dm]: did you mean force_reboot instead of revive here?

Comment (DeltaFire15) [reply code/modules/surgery/emergency_reboot.dm]: OOPS

Comment (DeltaFire15) [reply code/modules/surgery/emergency_reboot.dm]: fixed

**Pull request merged**